README.md

```diff
@@ -2,29 +2,74 @@
 
 A bookmarklet for monitoring GPT ad units and their associated targeting on web pages.
 
+* [Dev Requirements](#dev-requirements)
 * [Installation](#installation)
+* [Development](#development)
+* [Charles Setup](#charles-setup)
 * [Additional Information](#additional-information)
 * [Contributors](#contributors)
 
+## Dev Requirements
+- [Node](https://nodejs.org/en/)
+- [Charles](https://www.charlesproxy.com/)
```

Comment:
Perhaps, this says something like `A proxy server like [Charles](https://www.charlesproxy.com/)`

If devs have some other application they are tied into, they could use it instead.

Reply:
gotcha